src/main/java/org/jabref/model/groups/LastNameGroup.java

```diff
@@ -0,0 +1,57 @@
+package org.jabref.model.groups;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import org.jabref.model.entry.Author;
+import org.jabref.model.entry.AuthorList;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.Field;
+import org.jabref.model.strings.LatexToUnicodeAdapter;
+
+/**
+ * Matches based on a latex free last name in a specified field. The field is parsed as an author list and the last names are resolved of latex.
+ */
+public class LastNameGroup extends KeywordGroup {
+    public LastNameGroup(String groupName, GroupHierarchyType context, Field searchField, String lastName) {
+        super(groupName, context, searchField, LatexToUnicodeAdapter.format(lastName), true);
+    }
+
+    static List<String> getAsLastNamesLatexFree(Field field, BibEntry bibEntry) {
+        return bibEntry.getField(field).stream()
+                       .map(AuthorList::parse)
+                       .map(AuthorList::getAuthors)
+                       .flatMap(Collection::stream)
+                       .map(Author::getLastLatexFree)
+                       .flatMap(Optional::stream)
+                       .collect(Collectors.toList());
+    }
+
+    @Override
+    public boolean contains(BibEntry entry) {
+        return getAsLastNamesLatexFree(getSearchField(), entry).stream().anyMatch(name -> name.equals(getSearchExpression()));
+    }
+
+    @Override
+    public AbstractGroup deepCopy() {
+        return new LastNameGroup(getName(), getHierarchicalContext(), getSearchField(), getSearchExpression());
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        if (super.equals(other)) {
```

Comment:
You can easily use instanceof here that should also check for supertypes.
http://www.angelikalanger.com/Articles/JavaSolutions/SecretsOfEquals/Equals.html

And tip: Use Objects.equals(getSearchField, otherGroup.getSearchField)

Reply:
Objects.equals, good catch, I am not sure why I didn't use it.

If I understand you correctly, you'd like a specific check for `LastNameGroup` rather than relying on the `.getClass()` from `AbstractGroup.equals`
https://github.com/JabRef/jabref/blob/10180edf451c79785ec21a406b9bd309fe90ca80/src/main/java/org/jabref/model/groups/AbstractGroup.java#L52-L57
?

Reply:
if (this instanceof other) was the approach I thought of

Reply:
There were a few things in the article I had not encountered before, and I have made the changes I believe are necessary for a "class hierarchies of value types". (The code is now almost identical to "Listing 4" in the article)

Reply:
@Siedlerchr are you ok with the current approach?

Reply:
(By the way, Intellj can also auto-generate these equals/hash/tostring methods)